WebRoot/css/zm.css

```diff
@@ -7279,4 +7269,78 @@ input[type="radio"] + label {
 
 .ZmWhiteBlackViewNumberUsed {
 	@WhiteBlackListViewItemsNumberUsed@
-}
\ No newline at end of file
+}
+
+/* Preference signature interface panel */
+.ZmPrefSignaturePage {
+	@PrefSignaturePage@
+}
+
+.ZmPrefSignaturePage hr {
+	@PrefSignaturePageSeparator@
+}
+
+.ZmSignatureNameWrapper {
+	@PrefSignatureNameWrapper@
+}
+
+.ZmSignatureNameWrapper > label {
+	@PrefSignatureNameLabel@
+}
+
+.ZmSignatureNameWrapper .DwtInputField input[type='text'] {
```

Comment:
Maybe not. It not only focusses on width, but also has border from all edges as oppose to normal inputs where we have border from bottom only.